### ISSUE TITLE
Server ip's need updating

### ISSUE THREAD

**Issue by St0RM53** [FN-FAL113/cs2-server-picker#55]
They've changed again

**Comment by FN-FAL113** [comment]
![image](https://github.com/FN-FAL113/cs2-server-picker/assets/88238718/5487e561-f90e-4503-8186-afa7509410c8)
Hello storm, it should automatically get updated IP''s upon program execution. The message shown above will pop up everytime valve's makes a revision on their server data.

**Comment by St0RM53** [comment]
Yes i know this, but it's been some time since it last showed this message and there seem to be server changes.
Example usually Spain server for me is ~60ms ping, but now when i block it, in-game there is now Spain with 75ms ping that is unblocked.

Also blocking all servers except New York now, in-game it gives error message saying can't find any server, even though ping filtering is set to unlimited.

Btw is French servers only for connecting from within France? It doesn't ever show it in game.

**Comment by FN-FAL113** [comment]
> Yes i know this, but it's been some time since it last showed this message and there seem to be server changes. Example usually Spain server for me is ~60ms ping, but now when i block it, in-game there is now Spain with 75ms ping that is unblocked.
> 
> Also blocking all servers except New York now, in-game it gives error message saying can't find any server, even though ping filtering is set to unlimited.
> 
> Btw is French servers only for connecting from within France? It doesn't ever show it in game.

I do not recommend blocking most servers as it affects routing for possible matches. Second I do not manage the IP's and it is fetched by what valved provides, IP's are not manually inputted. Lastly Blocking servers may show different ping results in-game since the list in-game is harcoded and may use a different relay server instead.
